src/ChannelAdvisorAccess/REST/Services/ServiceBaseAbstr.cs

```
@@ -254,9 +254,7 @@ private async Task RefreshAccessTokenByRestCredentials( CancellationToken token,
 					}
 					catch( Exception responseEx )
 					{
-						ChannelAdvisorLogger.LogTrace(String.Format("Failed due to: '{0}', response: {1}", responseEx.Message, responseStr ) );
-						var channelAdvisorException = new ChannelAdvisorException( responseEx.Message, responseEx );
-						throw channelAdvisorException;

```

@maxim-saltanov-skuvault I am confused. You removed rethrowing. So now this "catch" will swallow the exception and it will not trigger a retry or failed sync, as it currently does in Prod. I thought we were trying to avoid changing the sync behavior, but just add logging. Am I misunderstanding something? Same in other places here.

---

@maxim-skuvault you're right, should be fixed for now